R/mmrm-methods.R

```diff
@@ -20,6 +20,10 @@
 #'    2. If the covariance structure is spatial, the covariance matrix of two time points with unit distance
 #'       will be displayed.
 #'
+#' `Anova` will return `anova` object with one row per variable and columns
+#' `Num Df`(numerator degrees of freedom), `Denom Df`(denominator degrees of freedom),
+#' `F Statistic` and `Pr(>=F)`.
+#'
```

Comment:
I don't want to export `Anova.mmrm` so no place to properly document it. so still put it here.